### PR TITLE
Added new listings, กัน

### PR DESCRIPTION
**Pull request by hashtagSELFIE**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,4 +87,6 @@
 - (ในคืน) แสงจันทร์ส่อง ระยิบตาดวงดารา ดั่งดวงตานางฟ้าจ้องมอง เป็นพยานในยามราตรีบรรเลงขับขาน เพลงรักเพื่อเรา [Cocktail feat. นิว นภัสสร - งานเต้นรำในคืนพระจันทร์เต็มดวง](https://youtu.be/y04gImu0qiw?t=122)
 - เรามาไกลเท่านี้ก็ดีเหลือเกิน ขอบใจนะที่เคยมีกันมันถึงเวลายอมรับความจริง [ฺBigAss - ลมเปลี่ยนทิศ](https://youtu.be/5H4Lg-rl58U?t=90)
 - (มี) จริงหรือ รักแรกพบเพียงสบตาแค่หนึ่งครั้ง แค่แรกเห็นเดินผ่านมาไม่พูดจา ไม่ทักไม่ทาย ไม่รู้ว่าใคร เหตุใดจึงรักกัน [Tattoo Colour - รักแรกพบ](https://youtu.be/rc7KnQAh_1I?t=36)
+- (เรา) กันเองอยู่แล้ว โธ่ เรื่องแค่นี้ไม่เห็นต้องปิด คนเราจะรัก ไม่ใช่เรื่องผิด บอกมาเถอะเธอคิดยังไง [กัน เดอะสตาร์ - กันเอง](https://youtu.be/MBWVdKl-W0s)
+...อ้างอิง https://www.siamzone.com/music/thailyric/180
```
Comment (narze) [inline README.md:91]: This line is not needed, also the link should include the timestamp krub 
image

**Review by narze — CHANGES_REQUESTED** [review]
Please fix the issue, by the way I'll add `hacktoberfest-accepted` to make this PR qualified before tomorrow 👍

**Comment by hashtagSELFIE** [comment]
Removed unnecessary lines and added timestamped

**Review by narze — APPROVED** [review]
👍

**Pull request merged**